src/preferences/dialog/dlgprefdeck.h

```diff
@@ -20,7 +20,14 @@ namespace TrackTime {
     enum class DisplayMode {
         Elapsed,
         Remaining,
-        ElapsedAndRemaining,
+        ElapsedAndRemaining
+    };
+
+    enum class DisplayFormat {
+        TRADITIONAL,
```

Comment:
Why do DisplayMode and DisplayFormat use inconsistent naming conventions for their enum literals?

Reply:
Good question. You're referring to the fact that one is in Sentence Case and the other in SNAKE_CASE, right?

Reply:
Of course, it's obvious ;)